modules/prebidServerBidAdapter/index.js

```diff
@@ -1045,19 +1041,19 @@ Object.assign(ORTB2.prototype, {
             if (!bidObject.vastUrl && bid.nurl) { bidObject.vastUrl = bid.nurl; }
           } else if (deepAccess(bid, 'ext.prebid.type') === NATIVE) {
             bidObject.mediaType = NATIVE;
-            let adm;
+            let ortb;
             if (typeof bid.adm === 'string') {
-              adm = bidObject.adm = JSON.parse(bid.adm);
+              ortb = bidObject.adm = JSON.parse(bid.adm);
             } else {
-              adm = bidObject.adm = bid.adm;
+              ortb = bidObject.adm = bid.adm;
             }
 
             let trackers = {
```

Comment:
Noticed this now (trying to mock a response for the PUC) - `trackers` does not seem to be used for anything?

Reply:
Correct, it's a leftover from the previous logic. What I decided to do here is to convert the two soon-deprecated properties `imptrackers` and `jstracker` into eventtrackers.